### ISSUE TITLE
Fair-Stockfish 14 in WinBoard: Illegal move "O-O" (rejected by first chess program)

### ISSUE THREAD

**Issue by numlockkey** [fairy-stockfish/Fairy-Stockfish#715]
Here  is the position: br2q1kr1n/ppp3pppp/4pp3c/1n1p6/1Q3P4/1P1P4P1/PBP1P1PPCb/1RNB2KR1N b HBhb - 1 1

I set Fairy-Stockfish 14 to play White. I try to castle manually by first clicking the King, and then the Rook. 

I get this error:

Illegal move "O-O" (rejected by first chess program)

The same move tested in Cute Chess works fine.

**Comment by ianfab** [comment]
Yes, Fairy-SF currently only understands the normal castling notation in the CECP/xboard protocol, not the FRC notation. The CECP documentation states that
> Castlings are indicated as the King move, e.g. e1g1 or e8c8, except for Fischer castling, which is O-O or O-O-O (oh, not zero!).

In UCI that all works fine, therefore other GUIs have no issue.
